internal/runners/protocol/protocol.go

```diff
@@ -60,5 +61,11 @@ func (p *Protocol) Run(params Params) error {
 		return locale.NewError("err_protocol_flag", "Invalid URL fragment, the only supported URL fragment is 'replace'")
 	}
 
-	return runbits.InvokeSilent("activate", fmt.Sprintf("--%s", parsed.Fragment), namespace.String())
+	// Execute state command
+	exe, err := os.Executable()
+	if err != nil {
+		return locale.WrapError(err, "err_protocol_os_executable", "Could not detect executable path of State Tool.")
+	}
+	_, _, err = osutils.ExecSimple(exe, []string{"activate", fmt.Sprintf("--%s", parsed.Fragment), namespace.String()}, nil)
```

Comment:
```suggestion
	_, _, err = osutils.ExecuteAndPipeStd(exe, []string{"activate", fmt.Sprintf("--%s", parsed.Fragment), namespace.String()},[]string{})
```

Sorry ExecSimple was a bad call on my end. InvokeSilent pipes os.Stdout and os.Stderr to the cmd, whereas ExecSimple does not. We want to make sure here that we still pipe stdout and stderr, otherwise the end-user won't see anything.

Reply:
Oh weird, maybe it's a Linux/macOS thing. I tested on Windows and it worked just fine. Regardless, I've made the change. Thanks for pointing it out.

Reply:
Huh, no idea what is happening on Windows. Strange..